components/DetailScreen/DetailHeader.js

```diff
@@ -157,13 +163,13 @@ const styles = StyleSheet.create({
     fontSize: 18,
     color: '#C4C4C4',
     fontWeight: 'bold',
-    fontFamily: "OpenSans-SemiBold",
+    fontFamily: 'OpenSans-SemiBold'
   },
   DetailButton: {
     fontSize: 18,
     color: 'black',
     fontWeight: 'bold',
-    fontFamily: "OpenSans-SemiBold",
+    fontFamily: 'OpenSans-SemiBold'
```

Comment:
I see this hapenning alot....I think it is from the autosavers with beautify on save lol (or prettier)